src/utils/urlUtil.ts

```diff
@@ -242,5 +229,14 @@ export const urlUtil = {
    */
   removeTrailingSlashes(url: string): string {
     return url.replace(/\/+$/, '');
+  },
+
+  getUrlRelativePath(url: string): string {
+    if (url.indexOf('://') > 0 || url.indexOf('//') === 0) {
```

Comment:
Is a URL starting with `//` considered valid when parsing?

Reply:
Yes, `new URL('//xxx')` throws `Invalid URL`
I can't find any test case for a URL starting with `//`. Even if I remove the second part of the `if` clause, all tests passed and the coverage is 100%.

I will remove the check `|| url.indexOf('//') === 0`